readme.md

```diff
@@ -671,6 +671,7 @@
 - [Open Source Photography](https://github.com/ibaaj/awesome-OpenSourcePhotography#readme)
 - [OpenGL](https://github.com/eug/awesome-opengl#readme) - Cross-platform API for rendering 2D and 3D graphics.
 - [GraphQL](https://github.com/chentsulin/awesome-graphql#readme)
+- [Urban Planning](https://github.com/APA-Technology-Division/planning-technology-resources#readme) - Resources & tools for urban planners & urbanists.
```

Comment:
The description here should succinctly describe what "urban planning" is, not what the list contains.

Reply:
Same with the description in the list itself: https://github.com/APA-Technology-Division/planning-technology-resources#planning-technology-resources--

Reply:
Whew...succinctly encapsulate urban planning...tall order, but I will review and revise accordingly. 

To clarify, the repo description must match what is in the awesome list? Or can they be just similar? I feel like the awesome list description should be shorter. 

If your time is short, I will just revise for review.

Reply:
> To clarify, the repo description must match what is in the awesome list? Or can they be just similar?

The one in the Awesome list should be succinct. The one in your list can be longer.